src/AuthenticatedAPIClient/axiosConfig.js

```diff
@@ -57,51 +56,13 @@ function applyAxiosInterceptors(authenticatedAPIClient) {
     return request;
   }
 
-  /**
-   * Ensure the browser has an unexpired JWT cookie before making API requests.
-   *
-   * This will attempt to refresh the JWT cookie if a valid refresh token cookie exists.
-   */
-  function ensureValidJWTCookie(request) {
-    const originalRequest = request;
-    const isAuthUrl = authenticatedAPIClient.isAuthUrl(originalRequest.url);
-    const accessToken = authenticatedAPIClient.getDecodedAccessToken();
-    const tokenExpired = authenticatedAPIClient.isAccessTokenExpired(accessToken);
-    if (isAuthUrl || !tokenExpired) {
-      return request;
-    }
-
-    if (!queueRequests) {
-      queueRequests = true;
-      authenticatedAPIClient.refreshAccessToken()
-        .then(() => {
-          queueRequests = false;
-          PubSub.publishSync(ACCESS_TOKEN_REFRESH, { success: true });
-        })
-        .catch((error) => {
-          // If no callback is supplied frontend-auth will (ultimately) redirect the user to login.
-          // The user is redirected to logout to ensure authentication clean-up, which in turn
-          // redirects to login.
-          if (authenticatedAPIClient.handleRefreshAccessTokenFailure) {
-            authenticatedAPIClient.handleRefreshAccessTokenFailure(error);
-          } else {
-            authenticatedAPIClient.logout();
-          }
-          PubSub.publishSync(ACCESS_TOKEN_REFRESH, { success: false });
-        });
+  function ensureValidJWTCookie(requestConfig) {
+    // if we've got what we need
```

Comment:
I don't understand this comment. Maybe better to speak to what `isAuthUrl` and why this translates to having what we need.  Or, does this translate to not needing anything?

Reply:
added `const jwtIsNotRequired = authenticatedAPIClient.isAuthUrl(axiosRequestConfig.url);`